Gruntfile.js

```diff
@@ -307,9 +311,11 @@ module.exports = function(grunt) {
 					[ WORKING_DIR + 'wp-includes/js/customize-preview.js' ]: [ './src/js/_enqueues/wp/customize/preview.js' ],
 					[ WORKING_DIR + 'wp-includes/js/customize-selective-refresh.js' ]: [ './src/js/_enqueues/wp/customize/selective-refresh.js' ],
 					[ WORKING_DIR + 'wp-includes/js/customize-views.js' ]: [ './src/js/_enqueues/wp/customize/views.js' ],
+					[ WORKING_DIR + 'wp-includes/js/edit-events.js' ]: [ './src/js/_enqueues/lib/edit-events.js' ],
```

Comment:
As mentioned below, I think this could be merged with `js/_enqueues/admin/post.js`